templates/squareup-javapoet/Project.cshtml

```diff
@@ -0,0 +1,99 @@
+<Project Sdk="Xamarin.Legacy.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>MonoAndroid12.0;net6.0-android31.0</TargetFrameworks>
+    <IsBindingProject>true</IsBindingProject>
+    <!--
+      No warnings for:
+       - CS0618: 'member' is obsolete: 'text'
+       - CS0109: The member 'member' does not hide an inherited member. The new keyword is not required
+       - CS0114: 'function1' hides inherited member 'function2'. To make the current method override that implementation, add the override keyword. Otherwise add the new keyword.
+       - CS0628: 'member' : new protected member declared in sealed class
+       - CS0108: 'member1' hides inherited member 'member2'. Use the new keyword if hiding was intended.
+       - CS0809: Obsolete member 'member' overrides non-obsolete member 'member'
+    -->
+    <NoWarn>0618;0109;0114;0628;0108;0809</NoWarn>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <AndroidClassParser>class-parse</AndroidClassParser>
+    <AndroidCodegenTarget>XAJavaInterop1</AndroidCodegenTarget>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Authors>Microsoft</Authors>
+    <Owners>Microsoft</Owners>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <PackageProjectUrl>https://go.microsoft.com/fwlink/?linkid=865435</PackageProjectUrl>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+
+    @foreach (var art in @Model.MavenArtifacts) 
+    {
+      <PackageId>@Model.NuGetPackageId</PackageId>
+      <RootNamespace>@Model.NuGetPackageId</RootNamespace>
+      <AssemblyName>@Model.NuGetPackageId</AssemblyName>
+      switch(@Model.NuGetPackageId)
+      {
+          case "Square.JavaPoet":
+            <Title>JavaPoet bindings for Xamarin.Android</Title>
+            <Summary>APIs to generate Java source code</Summary>
+            <Description>
+              JavaPoet is library, which provides APIs to generate Java source code. 
+              It can generate primitive types, reference types and their variants (such as classes, interfaces, 
+              enumerated types, anonymous inner classes), fields, methods, parameters, annotations, and Javadocs.
+            </Description>
+            <PackageDescription>
+              JavaPoet is library, which provides APIs to generate Java source code. 
+              It can generate primitive types, reference types and their variants (such as classes, interfaces, 
+              enumerated types, anonymous inner classes), fields, methods, parameters, annotations, and Javadocs.
+            </PackageDescription>
+            <PackageLicenseFile>LICENSE.md</PackageLicenseFile>
```

Comment:
As per all of the "wonderful" work with Component Governance, `@(PackageLicenseFile)` is "bad", and this should instead use `$(PackageLicenseExpression)`:

```xml
<PackageLicenseExpression>MIT</PackageLicenseExpression>
```

See also: https://github.com/mono/opentk/commit/daa9b2d577d398c95ca4014dd92c2a55fa67dcf5